### ISSUE TITLE
CRAN check still fails on tinytest.R

### ISSUE THREAD

**Issue by fitzLab-AL** [fitzLab-AL/gdm#49]
Hi @rvalavi - The package still fails during CRAN checks. Everything works fine for me locally when I build and check the package. I also don't get any errors when I run the tests.

Any ideas?

https://cran.r-project.org/web/checks/check_results_gdm.html

**Comment by rvalavi** [comment]
Hi @fitzLab-AL 

It's odd! I updated my `blockCV` package last week, and while it initially failed CRAN's auto checks on Windows, it still made it onto CRAN a few hours later. This could be an issue on their end.

I'm going to run some tests on those specific platforms using the `rhub` package to see if I can figure out why it's struggling to create the prediction raster layers.

**Comment by jarioksa** [comment]
@rvalavi it may not be so odd: the automatic tests are not updated very often, and it may take a week or so before they are updated after the fix. The test front page only gives the date when some of tests was last updated, and there may not be a way to see when each test was last run. There is no date in the test logs, but you can have a look at the version of R used in each.

**Comment by fitzLab-AL** [comment]
@rvalavi Any good news here? I am not able to replicate the errors shown on CRAN. The package will be removed on 12/8/24 if the issue is not addressed. Perhaps we can remove the problematic test for now?

**Comment by rvalavi** [comment]
Hi @fitzLab-AL , unfortunately, I think this is the solution! I also couldn't reproduce the error. 
I'll highlight those tests for now. Maybe we can solve it later.

**Comment by fitzLab-AL** [comment]
OK sounds like a plan @rvalavi. Hard to fix a problem that we can't replicate. I will send the new version of to CRAN soon.

**Comment by rvalavi** [comment]
Updated the tests in the #51. You can merge before submitting and it should be good to go.

**Comment by jarioksa** [comment]
It seems that CRAN release 1.6.0-4 also fails in tinytests, but now on `trans_raster`:

```
error in evaluating the argument 'x' in selecting a method for function 'global': object 'trans_raster' not found
```

Could the failure be related to this NOTE:
```
checking installed package size ... NOTE
  installed size is  5.1Mb
  sub-directories of 1Mb or more:
    extdata   3.8Mb
```
With this respect, issue #52 may also be relevant.

**Comment by rvalavi** [comment]
Yes, I think we need to remove all the rester test for now! I added the changes in a new pull request (#54). 
I recommend the new submission to CRAN be directly v1.6.0-5 (#53) rather than v1.9.0.-4, if possible.
